src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -30,32 +28,14 @@ namespace OpenTelemetry.Exporter
 {
     internal static class OtlpExporterOptionsExtensions
     {
-#if NETSTANDARD2_1 || NET6_0_OR_GREATER
         public static GrpcChannel CreateChannel(this OtlpExporterOptions options)
-#else
-        public static Channel CreateChannel(this OtlpExporterOptions options)
-#endif
         {
             if (options.Endpoint.Scheme != Uri.UriSchemeHttp && options.Endpoint.Scheme != Uri.UriSchemeHttps)
             {
                 throw new NotSupportedException($"Endpoint URI scheme ({options.Endpoint.Scheme}) is not supported. Currently only \"http\" and \"https\" are supported.");
             }
 
-#if NETSTANDARD2_1 || NET6_0_OR_GREATER
             return GrpcChannel.ForAddress(options.Endpoint);
```

Comment:
OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests.OtlpExporterOptionsExtensionsTests.GetTraceExportClient_SupportedProtocol_ReturnsCorrectExportClient(protocol: Grpc, expectedExportClientType: typeof(OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient.OtlpGrpcTraceExportClient))
   Source: OtlpExporterOptionsExtensionsTests.cs line 91
   Duration: 1.2 sec

  Message: 
System.PlatformNotSupportedException : gRPC requires extra configuration on .NET implementations that don't support gRPC over HTTP/2. An HTTP provider must be specified using GrpcChannelOptions.HttpHandler.The configured HTTP provider must either support HTTP/2 or be configured to use gRPC-Web. See https://aka.ms/aspnet/grpc/netstandard for details.

  Stack Trace: 
HttpHandlerFactory.CreatePrimaryHandler()
GrpcChannel.CreateInternalHttpInvoker(HttpMessageHandler handler)
GrpcChannel.ctor(Uri address, GrpcChannelOptions channelOptions)
GrpcChannel.ForAddress(Uri address, GrpcChannelOptions channelOptions)
GrpcChannel.ForAddress(Uri address)
OtlpExporterOptionsExtensions.CreateChannel(OtlpExporterOptions options) line 39
OtlpGrpcTraceExportClient.ctor(OtlpExporterOptions options, TraceServiceClient traceServiceClient) line 39
OtlpExporterOptionsExtensions.GetTraceExportClient(OtlpExporterOptions options) line 81
OtlpExporterOptionsExtensionsTests.GetTraceExportClient_SupportedProtocol_ReturnsCorrectExportClient(OtlpExportProtocol protocol, Type expectedExportClientType) line 106